tests/welcome.test.js

```diff
@@ -0,0 +1,21 @@
+const chai = require('chai');

```

Comment:
Could you use `import` syntax from ES6 ?

Reply:
> Could you use `import` syntax from ES6 ?

the project is still in ES5 syntax and as you know @Muhire-Josue is the one who is working on setting up babel ES6